README.md

```diff
@@ -5,23 +5,251 @@
 
 # Istio Operator
 
-Update 4/22/19
-
-This repo is for the initial, alpha version of the official Istio install operator. It's currently only open to the core 
-design team for commits, while we merge prototypes that evolved in parallel. Once we have
-a basic functional skeleton in place we'll open the repo up for community contributions.
-
-The initial goals/MVP for the operator are:
-
-1.   Replace helm tiller as the recommended install upgrade controller. We will continue to use helm templates and
-the helm library to generate manifests under the hood, but will take ownership of the logic behind the install API.
-The operator will have modes of operation analogous to tiller - controller in and out of cluster and one-shot manifest
-generation for use with external toolchains.
-1.   Provide a structured API to eventually replace the de-facto values.yaml API. This API will be partially moved
-to CRDs used to configure Istio components directly, hence the operator API will only contain the portions that will 
-remain in the operator CRD, and continue to use values.yaml for the parameters that will be moved to component CRDs.
-The API will be aligned with Istio a-la-carte/building blocks philosophy and have a functional structure.
-1.   Add an overlay mechanism to allow users to customize the rendered manifest in a way that survives upgrades. 
-1.   Support upgrade, both in-place and dual control plane version rollover. The latter will be supported through the 
-use of the [istio/installer](https://github.com/istio/installer) charts.
-1.   Provide validation of all parameters.
+Istio operator is not yet ready for users to try out (it will be shortly, stay tuned on discuss.istio.io). 
+However, the code is stable enough for developers to evaluate and experiment with. Contributions are very welcome - 
+see [open issues](), create a feature proposal, or come to the weekly Environments Working Group meeting if you have ideas. 
+
+### Background
+
+The current [helm installation parameters](https://istio.io/docs/reference/config/installation-options/) have been reorganized into two groups:
+
+1.  A new [platform level installation API](https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto), dealing with k8s settings like resources, auto scaling, pod disruption budgets etc. 
+1.  A configuration API, dealing with Istio control plane configuration settings. This API currently uses the [helm installation parameters](https://istio.io/docs/reference/config/installation-options/) for backwards compatibility, but will be reorganized and slimmed down in the future.
+
+Some parameters will therefore temporarily be in both APIs - for example, setting k8s resources can be done through either API above. However, it's recommended to use the first API, since this has a more consistent structure and it is the one that will remain going forward.  
```

Comment:
please remove forward looking statements.
A better way to state this would be "The Istio community recommends usage of the first API as this API will naturally follow the graduation process for APIs while the configuration API is planned for deprecation.